### PR TITLE
Fix capstone_test.c #2059

### LINKED ISSUE #2058: [CIFuzz] name 'CS_OP_INVALID' is not defined in tricore_const
https://github.com/capstone-engine/capstone/actions/runs/5352038108/jobs/9706549501?pr=2057#step:4:19147

```shell
+ cd /src/capstonenext/suite
+ mkdir fuzz/corpus
+ find MC/ -name '*.cs'
+ ./test_corpus.py
Traceback (most recent call last):
  File "./test_corpus.py", line 5, in <module>
    from capstone import *
  File "build/bdist.linux-x86_64/egg/capstone/__init__.py", line 386, in <module>
  File "build/bdist.linux-x86_64/egg/capstone/tricore.py", line 5, in <module>
  File "build/bdist.linux-x86_64/egg/capstone/tricore_const.py", line 2, in <module>
NameError: name 'CS_OP_INVALID' is not defined
2023-06-23 01:56:14,725 - root - ERROR - Building fuzzers failed.
```

Cause by commit https://github.com/capstone-engine/capstone/commit/f66d2cb9870b5ca509568e4bae883203b37e8934

```
TRICORE_OP_INVALID = 0
TRICORE_OP_REG = 1
TRICORE_OP_IMM = 2
TRICORE_OP_MEM = 3

# TriCore registers
TRICORE_OP_INVALID = CS_OP_INVALID
TRICORE_OP_REG = CS_OP_REG
TRICORE_OP_IMM = CS_OP_IMM
TRICORE_OP_MEM = CS_OPT_MEM
TRICORE_OP_COUNT = 8
```

### LINKED ISSUE #2059: [Test] issues.cs Segmentation fault(11) when using make compile
https://github.com/capstone-engine/capstone/actions/runs/5117561940/jobs/9200760293#step:8:1015

```
[-] Detailed report for issues.cs:

	[+] Unknown test: 
		Test failed with exception: Segmentation fault(11)

	[+] Unknown test: 
		Test failed with exception: Segmentation fault(11)
...
```

Cause by commit https://github.com/capstone-engine/capstone/commit/6c3a899646b02ff9030cfd38ccad068b4f2ba1eb

### PR DESCRIPTION
**Pull request by b1llow**
closes #2059
closes #2058

### CODE REVIEW

**Comment by b1llow** [comment]
@kabeor take a took please.

**Comment by kabeor** [comment]
Great! Thanks for your time:)

**Comment by kabeor** [comment]
```
+ cd /src/capstonenext/suite
+ mkdir fuzz/corpus
+ find MC/ -name '*.cs'
+ ./test_corpus.py
Traceback (most recent call last):
  File "./test_corpus.py", line 5, in <module>
    from capstone import *
  File "build/bdist.linux-x86_64/egg/capstone/__init__.py", line 388, in <module>
  File "build/bdist.linux-x86_64/egg/capstone/tricore.py", line 5, in <module>
  File "build/bdist.linux-x86_64/egg/capstone/tricore_const.py", line 1, in <module>
  File "build/bdist.linux-x86_64/egg/capstone/__init__.py", line 388, in <module>
ImportError: cannot import name tricore
```

File: bindings/python/capstone/__init__.py [hunk 1]
```diff
@@ -110,6 +110,8 @@
     'CS_OPT_ON',
     'CS_OPT_OFF',
 
+    'CS_OPT_MEM',
```
Comment (Rot127) [inline bindings/python/capstone/__init__.py:113]: Shouldn't it be `CS_OP_MEM`?

Comment (b1llow) [reply bindings/python/capstone/__init__.py:113]: Just saw that this has been fixed in your autosync branch.

My mistake, I was too careless.

**Comment by Rot127** [comment]
Don't worry. Your version is better. I'll cherry-pick it.

23 Jun 2023 17:53:08 billow ***@***.***>:

**Comment by kabeor** [comment]
Nice, Merged.

**Pull request merged**